Extension/c_cpp_properties.schema.json

```diff
@@ -58,6 +58,11 @@
                             "type": "string"
                         }
                     },
+                    "windowsSdkVersion": {
+                        "description": "Version of the Windows SDK include path to use on Windows, e.g. '10.0.17134.0'.",
+                        "type": "string",
+                        "pattern": "^\\d{2}\\.\\d{1}\\.\\d{5}\\.\\d{1}$|^8.1$"
```

Comment:
The 8.1 part should have a backslash before the dot

Reply:
Ooops...